src/Snppts/Snippets/PointOfSale.cs

```diff
@@ -0,0 +1,25 @@
+public class PointOfSale : IAmASnippet
+{
+    public string Slug => "point-of-sale";
+    public string Title => "Point of Sale";
+    public string GithubRepoName => "snpptsio/Snppts.PointOfSale";
```

Comment:
Don't think this is right? That's also why the build fails

```suggestion
    public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("dotnet/maui-samples/tree/main/6.0/Apps/PointOfSale");
```

Not sure if the above will work though, don't think we really took into account subfolders on a repo 😅